<launch>
    <node pkg="map_server" type="map_server" name="map_server_node" args="$(find extract_globalpath)/map_file/map.yaml"/>
    <node pkg="extract_globalpath" type="extract_globalpath_node" name="extract_globalpath_node" output="screen">
        <param name="path_file" value="$(find extract_globalpath)/path_file/path.csv"/>
        <param name="weight_data" value="0.47" />
        <param name="weight_smooth" value="0.2" />
        <param name="tolerance" value="0.1" />
    </node>
</launch>
